ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3503,6 +3503,28 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-ui
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack
+- as: vsphere-ipi-ovn-dualstack-privmaryv6-f28
+  cron: 35 12 7 * *
+  steps:
+    cluster_profile: vsphere
```

Comment:
I remember that we just migrated `vsphere` cluster_profile to `vsphere-2`, so I think we also need to change it to `vsphere-2` here.

Reply:
probably need to re-run a pj-rehearse then.